### PR TITLE
SCRUM-2294 add standard and generic-control experiments as default va…

### PR DESCRIPTION
**Pull request by cmpich**
…lues for ZFIN annotations.

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java [hunk 1]
```diff
@@ -37,6 +37,9 @@
 @AGRCurationSchemaVersion(min = "1.2.0", max = LinkMLSchemaConstants.LATEST_RELEASE, dependencies = { ExperimentalCondition.class })
 public class ConditionRelation extends UniqueIdAuditedObject {
 
+	public static final String STANDARD = "Standard";
+	public static final String GENERIC_CONTROL = "Generic-control";
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java]: Can these go into the constants class?

Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java]: I prefer having those constants on the class they belong to.

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java]: Are they used outside the class at all? If not change them to private.

Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java]: I put them into an inner class of ConditionRelation. Constant which then makes it obvious what this constant is for and that it is a constant.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**